### PR TITLE
Selenium: Implement usecase which covers opening a file from libraries after refreshing

### PR DESCRIPTION
**Pull request by Ohrimenko1988**
### What does this PR do?
Implement usecase which covers opening a file from libraries after refreshing

### What issues does this PR fix or reference?
Issue: https://github.com/eclipse/che/issues/9992

### CODE REVIEW

**Review by SkorikSergey — APPROVED** [review]

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/projectexplorer/dependencies/OpenExternalLibraryFileAfterRefreshTest.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    seleniumWebDriver.navigate().refresh();
+
+    // check that after browser refresh is still possible to open the same file
+    projectExplorer.waitProjectExplorer();
+    projectExplorer.waitItem(PROJECT_NAME);
+    openFileFromExternalLibraryAndCheckDisplayedText();
+  }
+
+  private void openFileFromExternalLibraryAndCheckDisplayedText() {
+    projectExplorer.waitVisibilityByName("README.md");
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/projectexplorer/dependencies/OpenExternalLibraryFileAfterRefreshTest.java]: We could shorten a code if we will have the method:
`projectExplorer.waitVisibilityByName(String... names)`
`projectExplorer.openItemByVisibleNameInExplorer(String... names)`

**Comment by dmytro-ndp** [comment]
Don't forget to add the test `OpenExternalLibraryFileAfterRefreshTest` to the CheSuite.xml.

File: selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/projectexplorer/dependencies/OpenExternalLibraryFileAfterRefreshTest.java (hunk 1 above)
Comment (Ohrimenko1988) [reply selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/projectexplorer/dependencies/OpenExternalLibraryFileAfterRefreshTest.java]: The method "projectExplorer.openItemByVisibleNameInExplorer(String... names)" already has the "projectExplorer.waitVisibilityByName(String... names)" method inside. 
In this particular case checks node expanding, not displaying element before clicking.
```
projectExplorer.waitVisibilityByName("README.md");
projectExplorer.waitVisibilityByName("pom.xml");
projectExplorer.waitVisibilityByName("External Libraries");
```

**Review by dmytro-ndp — APPROVED** [review]

File: selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/ProjectExplorer.java [hunk 2]
```diff
@@ -576,6 +591,18 @@ public void openItemByVisibleNameInExplorer(String name) {
     loader.waitOnClosed();
   }
 
+  /**
+   * Invokes the {@link ProjectExplorer#openItemByVisibleNameInExplorer(String)} method for each
+   * specified {@code name}.
```
Comment (dmytro-ndp) [inline selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/ProjectExplorer.java]: You may mean `{@code names}`.

**Pull request merged**